pkg/apis/deviceplugin/v1/qatdeviceplugin_webhook.go

```diff
@@ -91,5 +91,28 @@ func (r *QatDevicePlugin) validatePlugin() error {
 		}
 	}
 
+	if len(r.Spec.ProvisioningConfig) > 0 {
+		if len(r.Spec.InitImage) == 0 {
+			return errors.Errorf("ProvisioningConfig is set with no InitImage")
+		}
+
+		// check if 4xxxvf is enabled
+		contains4xxxvf := false
```

Comment:
the name of this variable makes no sense if we add new devices to `devicesWithCapabilities` some day. It was OK already but it got changed